File: OEM/Domain/ExecutedOperations/ExecutedOperationDto.cs
using System;

namespace OEMAPI.Domain.ExecutedOperations
{
    public class ExecutedOperationDto
    {
        public Guid Id { get; set; }
        public string VveId { get; set; }
        public Guid PlannedOperationId { get; set; }
        public string ResourceId { get; set; }
        public string StaffId { get; set; }
        public DateTime ActualStart { get; set; }
        public DateTime? ActualEnd { get; set; }
        public string Status { get; set; }
    }
}

File: OEM/src/dto/ExecutedOperationDTO.ts
export interface ExecutedOperationDTO {
  id: string;
  vesselVisitExecutionId: string;
  plannedOperationId: string;
  operationPlanId?: string;
  resourceId: string;
  staffId: string;
  actualStart: string;
  actualEnd?: string;
  status: string;
  syncStatus?: string;
}

export interface CreateExecutedOperationDTO {
  vesselVisitExecutionId: string;
  plannedOperationId: string;
  operationPlanId?: string;
  resourceId: string;
  staffId: string;
  actualStart: string;
  actualEnd?: string;
  status: string;
  syncStatus?: string;
}

export interface UpdateExecutedOperationDTO {
  actualStart?: string;
  actualEnd?: string;
  status?: string;
  syncStatus?: string;
}
